docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md

```diff
@@ -0,0 +1,40 @@
+# 字符比较函数和运算符
+
+| 名称      | 描述                 |
+|:-----    | :------------------: |
+| CONTAINS | 搜索包含指定字段的字符串，大小写敏感 |
+
+* CONTAINS
+
+`CONTAINS` 运算符用来搜索包含指定字段的字符串，且大小写敏感。`CONTAINS` 运算符左右两侧必须为字符串类型。
+
+```ngql
+nebula> GO FROM 107 OVER serve WHERE $$.team.name CONTAINS "riors" \
```

Comment:
You can add an example to use  insensitive  case 

```
nebula> GO FROM 107 OVER serve WHERE upper($$.team.name) CONTAINS "WARR" \
               YIELD $^.player.name, serve.start_year, serve.end_year, $$.team.name;
```

Reply:
good point 3q